client/omnichannel/directory/contacts/contextualBar/ContactInfo.js

```diff
@@ -79,6 +80,11 @@ export function ContactInfo({ id }) {
 		return false;
 	};
 
+	const onChatHistory = () => {
+		const { _id } = lastChat;
+		FlowRouter.go(`/live/${ _id }/contact-chat-history`);
```

Comment:
This is not the correct route to call here.
It should be something like:

`FlowRouter.go(`/contact/info/${ contact-id }/contact-chat-history`);`